cfgov/unprocessed/css/pages/external-site.less

```diff
@@ -3,33 +3,6 @@
    Third-Party links pages at /external-site/.
    ========================================================================== */
 
-.external-site_container {
-    .content_bar {
-        background: @gray-40;
-        height: 1px;
-    }
-
-    .padded-container {
-        background-color: @white;
-    }
-}
-
-.external-site_text {
-    font-weight: 500;
-}
-
-.external-site_back-btn {
-    &:link,
-    &:visited {
-        background-color: @dark-gray;
-    }
-
-    &:hover {
-        background-color: lighten(@dark-gray, 10%);
-    }
-}
-
-.external-site_back-btn,
 .external-site_proceed-btn {
```

Comment:
This entire set of properties now does nothing and can be removed.

Reply:
Removed!